end_to_end_tests/golden-record/my_test_api_client/api/responses/reference_response.py

```diff
@@ -0,0 +1,122 @@
+from http import HTTPStatus
```

Comment:
This file is just an example of what the generated code looks like - there's a test that compares it to the output.